### ISSUE TITLE
Niemożliwość znalezienia kategorii przez API

### ISSUE THREAD

**Issue by pgasidlo** [allegro/allegro-api#8899]
Występuje problem: 
 - [X] z zasobem REST API / with a REST API resource
 - [X] ze środowiskiem produkcyjnym / with the production environment

Na Allegro jest kategoria "Zabawki":

https://allegro.pl/kategoria/zabawki-11818

Pod tą kategorią jest podpięta kategoria "Gry towarzyskie":

https://allegro.pl/kategoria/zabawki-gry-towarzyskie-323046

Pobierając dane metodą:

```GET sale/categories?parent.id=11818``` 

w odpowiedzi dostaję 20 podkategorii. Mam wszystkie co na stronie Allegro - ale wśród nich nie mam 323046.

Gdy pytam o tą kategorię przez:

```GET sale/categories/323046```

dostaję błąd informujący, że kategoria nie istnieje:

```
object(stdClass)#162 (6) {
  ["code"]=>
  string(5) "ERROR"
  ["message"]=>
  string(27) "Category '323046' not found"
  ["details"]=>
  string(25) "ResourceNotFoundException"
  ["path"]=>
  NULL
  ["userMessage"]=>
  string(27) "Category '323046' not found"
  ["metadata"]=>
  object(stdClass)#163 (0) {
  }
}
```

**Comment by MaciejFrackowiak** [comment]
Jest to kategoria nawigacyjna, czyli dostępna tylko podczas przeglądania ofert na Allegro. W drzewie asortymentowanym, czyli tam gdzie sprzedawca wystawia ofertę jest ona niedostępna - czyli API zwraca właściwe dane.

**Comment by MaciejFrackowiak** [comment]
Z punktu widzenia sprzedającego interesują Cię tylko kategorie asortymentowe, które są dostępne w drzewie kategorii i parametrów. Wszelkie ewentualne mapowania pozostają wyłącznie w Twojej gestii.

**Comment by pgasidlo** [comment]
Z punktu widzenia sprzedającego interesuje mnie również konkurencja.

**Comment by MaciejFrackowiak** [comment]
Rozumiem, jednak założenia /offers/listing są inne - oddanie listingu Allegro w API.

**Comment by pgasidlo** [comment]
OK. To jedno pytanie. Każda z kategorii (nawigacyjna, asortymentowa) jest oznaczona unikalną liczbą (dobra, są jakieś UUIDy, ale tylko na głównych drzewach). Czy mozliwy jest przypadek, ze *ta sama* liczba jest używana w kategorii nawigacyjnej i kategorii asortymentowej i opisuje całkiem inną kategorię (tj. np. "cześci samochodowe" w nawigacyjnej i "książki dla dzieci" w asortymentowej)?

**Comment by MaciejFrackowiak** [comment]
Nie, każda posiada osobny identyfikator.

**Comment by lwadas** [comment]
Dzień dobry,

Czy w dokumentacji REST API jest coś wspomniane o tym że istnieje takie coś jak kategoria nawigacyjna?
Na jakiej zasadzie oferty są przydzielane lub nie przydzielane do kategorii nawigacyjnych?

**Comment by MaciejFrackowiak** [comment]
@lwadas Nie, ponieważ z punktu widzenia API drzewo kategorii i parametrów jest jedno - to wg niego sprzedający wystawia oferty. Ogólne informacje dostępne są w [Pomocy Allegro](https://help.allegro.com/sell/pl/a/jak-laczymy-kategorie-w-ktorych-wystawiasz-oferty-5LaEYwWw2H5).
